test/e2e_node/pod_host_ips.go

```diff
@@ -43,7 +43,7 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = common.SIGDescribe("Pod Host IPs", framework.WithSerial(), func() {
```

Comment:
Please mark it as NodeConformance. Otherwise it may not run in any of test lanes